datamade/just-spaces
# Rename 'justspacesapp' to 'surveys'

## Overview

Rename the backend survey-collector app from `justspacesapp` to `surveys`.

### Notes

Unfortunately, doing this kind of app renaming can seriously mess with migrations. See: https://gist.github.com/jamesmfriedman/6168003

The way I got around this was simply by dropping and recreating the database following the instructions in the README. If we need to preserve data in the staging database, we can do something a little bit more sophisticated (dump out relevant data and reimport it), but it'd be nice to be able to skip that and save time if we can.

## Testing Instructions

 * Drop and recreate your database following the instructions in the README
 * Run `python manage.py test` and confirm that tests pass
 * Run `python manage.py runserver` and confirm that the app loads properly at localhost:8000
 * Run `grep -iRF "justspacesapp" --exclude-dir .git .` and confirm there are no references to the old app name in the code 

Connects #55
